src/main/java/com/atlassian/bitbucket/jenkins/internal/provider/JenkinsProvider.java

```diff
@@ -15,4 +15,12 @@
      */
     @Nonnull
     Jenkins get();
+
+    /**
+     * Returns the Jenkins root URL. This method should never be invoked from inside the extensions as the underlying
+     * implementation iterates over exisiting extensions and thus creating circular dependency.
```

Comment:
? this seems weird, because the `Jenkins.get().getRootUrl()` does not do that?

Reply:
I have fixed the typo. This should never be invoked from Extension constructor as it causes cyclic dependency loop.